examples/compiled/bar_aggregate_count.svg

```diff
@@ -1 +1 @@
line></g><g class="mark-text role-axis-title" style="pointer-events: none;"><text text-anchor="middle" transform="translate(100,31)" style="font: bold 11px sans-serif; fill: #000; opacity: 1;">BIN(precipitation)</text></g></g></g></g><g class="mark-group role-axis"><g transform="translate(0.5,0.5)"><path class="background" d="M0,0h0v0h0Z" style="pointer-events: none; fill: none;"></path><g><g class="mark-rule role-axis-tick" style="pointer-events: none;"><line transform="translate(0,200)" x2="-5" y2="0" style="fill: none; stroke: #888; stroke-width: 1; opacity: 1;"></line><line transform="translate(0,171)" x2="-5" y2="0" style="fill: none; stroke: #888; stroke-width: 1; opacity: 1;"></line><line transform="translate(0,143)" x2="-5" y2="0" style="fill: none; stroke: #888; stroke-width: 1; opacity: 1;"></line><line transform="translate(0,114)" x2="-5" y2="0" style="fill: none; stroke: #888; stroke-width: 1; opacity: 1;"></line><line transform="translate(0,86)" x2="-5" y2="0" style="fill: none; stroke: #888; stroke-width: 1; opacity: 1;"></line><line transform="translate(0,57)" x2="-5" y2="0" style="fill: none; stroke: #888; stroke-width: 1; opacity: 1;"></line><line transform="translate(0,29)" x2="-5" y2="0" style="fill: none; stroke: #888; stroke-width: 1; opacity: 1;"></line><line transform="translate(0,0)" x2="-5" y2="0" style="fill: none; stroke: #888; stroke-width: 1; opacity: 1;"></line></g><g class="mark-text role-axis-label" style="pointer-events: none;"><text text-anchor="end" transform="translate(-7,203)" style="font: 10px sans-serif; fill: #000; opacity: 1;">0</text><text text-anchor="end" transform="translate(-7,174.42857142857144)" style="font: 10px sans-serif; fill: #000; opacity: 1;">200</text><text text-anchor="end" transform="translate(-7,145.85714285714286)" style="font: 10px sans-serif; fill: #000; opacity: 1;">400</text><text text-anchor="end" transform="translate(-7,117.28571428571429)" style="font: 10px sans-serif; fill: #000; opacity: 1;">600</text><text text-anchor="end" transform="translate(-7,88.71428571428572)" style="font: 10px sans-serif; fill: #000; opacity: 1;">800</text><text text-anchor="end" transform="translate(-7,60.14285714285714)" style="font: 10px sans-serif; fill: #000; opacity: 1;">1,000</text><text text-anchor="end" transform="translate(-7,31.571428571428584)" style="font: 10px sans-serif; fill: #000; opacity: 1;">1,200</text><text text-anchor="end" transform="translate(-7,3)" style="font: 10px sans-serif; fill: #000; opacity: 1;">1,400</text></g><g class="mark-rule role-axis-domain" style="pointer-events: none;"><line transform="translate(0,200)" x2="0" y2="-200" style="fill: none; stroke: #888; stroke-width: 1; opacity: 1;"></line></g><g class="mark-text role-axis-title" style="pointer-events: none;"><text text-anchor="middle" transform="translate(-50,100) rotate(-90) translate(0,-2)" style="font: bold 11px sans-serif; fill: #000; opacity: 1;">Number of Records</text></g></g></g></g></g></g></g></g></svg>
```

Comment:
@domoritz  I wonder if your result is different from mine on this one.

Reply:
<img width="269" alt="screen shot 2017-10-15 at 18 18 21" src="https://user-images.githubusercontent.com/589034/31591511-44760520-b1d5-11e7-8299-570fcaeb5be4.png">